README.md

```diff
@@ -118,49 +92,28 @@ The division `title` attribute is optional.  If this is not present the slug wil
 
 ### Running the application
 
-If you are [using docker](https://github.com/alphagov/govuk-docker) to run the application (which is advised) it will be available on the host at http://frontend.dev.gov.uk/
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
-To run the application standalone, run [static](https://github.com/alphagov/static) and execute the following command:
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+
+If you are using the `startup.sh` script, first run [static]()https://github.com/alphagov/static) and execute the following command:
```

Comment:
Just a fly-by comment in case you want to fix: this is incorrect, since `./startup.sh --live` will point at the live instance of Static, so you don't need it running locally (or to set the corresponding environment variable).